client/layout/guided-tours/tours/simple-payments-tour.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Link,
+	Quit,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNotNewUser,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const SimplePaymentsTour = makeTour(
+	<Tour
+		name="simplePaymentsTour"
+		version="20170816"
+		path="/post/"
+		when={ and( isDesktop, isNotNewUser ) }
+	>
+		<Step
+			name="init"
+			arrow="top-left"
+			target=".editor-html-toolbar__button-insert-media, .mce-wpcom-insert-menu button"
+			placement="below"
+			style={ { marginLeft: '22px', zIndex: 'auto' } }
+		>
+			<p>
+				{
+					translate( 'Did you know? ' +
+						'If your site is on the Premium or Business plan, you can add {{strong}}payment buttons{{/strong}} here!', {
+							components: {
+								strong: <strong />,
+							}
+						}
+					)
+				}
+			</p>
+			<ButtonRow>
+				<Quit primary>
+					{ translate( 'Got it, sounds good!' ) }
+				</Quit>
+			</ButtonRow>
+			<Link href="https://en.support.wordpress.com/simple-payments">
+				{ translate( 'Learn more about Simple Payments.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Learn more about Sitemaps.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).